<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_http</class>
 <widget class="QDialog" name="Dialog_http">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="get_btn">
       <property name="text">
        <string>Get</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Url:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="getUrl_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="post_btn">
       <property name="text">
        <string>Post</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Url:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="postUrl_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Data:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="postData_lineEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
